contrib/adsk/libraries/adsklib/adsklib_legacy_ng.mtlx

```diff
@@ -516,6 +516,497 @@
     <output name="output_color3" type="color3" xpos="21.615942" ypos="0.560345" nodename="mix_colors" />
   </nodegraph>
 
+  <nodegraph name="NG_util_specklenoise_float" nodedef="ND_util_specklenoise_float">
```

Comment:
We probably need a readme in future to document the math used here.

Reply:
The RRT code just say "taken from the Max code".
Basically the noise is multiplied by 1.65, clamped, and the range adjusted. No idea why.